java/api/src/main/java/io/ray/api/options/ActorLifetime.java

```diff
@@ -1,9 +1,9 @@
 package io.ray.api.options;
 
-/** The enumeration class is used for declaring lifetime of actors. It's non detached by default. */
+/** The enumeration class is used for declaring lifetime of actors. */
 public enum ActorLifetime {
-  DEFAULT("DEFAULT", 0),
-  DETACHED("DETACHED", 1);
+  DETACHED("DETACHED", 0),
+  NON_DETACHED("NON_DETACHED", 1);
```

Comment:
Will this be a breaking change of Java API?

Reply:
Yes. But `DEFAULT` was introduced not long ago. There is no official release version includes it currently. But it will be  introduced in ray1.1.10.  So I believe it's not a big problem.